src/content/guides/author-libraries.md

```diff
@@ -324,7 +324,7 @@ You can expose the library in the following ways:
 - Variable: as a global variable made available by a `script` tag (`libraryTarget:'var'`).
 - This: available through the `this` object (`libraryTarget:'this'`).
 - Window: available through the `window` object, in the browser (`libraryTarget:'window'`).
-- UMD: available after AMD or CommonJS `require` (`libraryTarget:'umd'`).
+- UMD: available under all the module definitions (`libraryTarget:'umd'`).
```

Comment:
Can you clarify this `all the module definitions`? Does it include es module? As far as I know, umd would only work with AMD, CommonJS and global variable.

Reply:
see https://webpack.js.org/configuration/output/#outputlibrarytarget This exposes your library under all the module definitions, allowing it to work with CommonJS, AMD and as global variable. Take a look at the UMD Repository to learn more.

Reply:
That one would be wrong too. UMD doesn't include ESM, webpack has this `output.library.type: 'module'` to output an ESM library.